pkg/scheduler/framework/plugins/volumezone/volume_zone.go

```diff
@@ -227,6 +241,10 @@ func (pl *VolumeZone) Filter(ctx context.Context, cs *framework.CycleState, pod
 
 	for _, pvTopology := range podPVTopologies {
 		v, ok := node.Labels[pvTopology.key]
+		if !ok && isBetaLabel(pvTopology.key) {
```

Comment:
I think this would be enough?
```suggestion
		if !ok {
```

Reply:
Hmmm. Yes, this will make the code cleaner🤔️.